### PR TITLE
Add script for question answering (SQuAD) accuracy evaluation of BERT model

### PR DESCRIPTION
**Pull request by tianleiwu**
**Description**: 

Add script to evaluate accuracy of BERT/DistilBERT/Roberta models on question-answering task. 

By default, pretrained model `bert-large-uncased-whole-word-masking-finetuned-squad` will be used if model name is not specified. If onnx path is not specified, optimum will be used to export an ONNX model for testing.

Example usage:

* Evaluate with CPU execution provider:
`python eval_squad.py`

* Evaluate with CUDA execution provider:
`python eval_squad.py --use_gpu`

* Evaluate an optimized onnx model for 'distilbert-base-cased-distilled-squad' with sequence lengths 128/192/256/384 on first 100 samples:
`python eval_squad.py -m distilbert-base-cased-distilled-squad --use_gpu -s 128 192 256 384 --onnx_path ./optimized_fp16.onnx -t 100`

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

Evaluate the impact of kernel changes on accuracy.

### CODE REVIEW

**Comment by snnn** [comment]
So, why don't put such things in https://github.com/microsoft/onnxruntime-inference-examples ? Every file in this repo is subject to strict compliance rules, but that one doesn't because we don't ship any package from that example repo.

**Comment by tianleiwu** [comment]
> So, why don't put such things in https://github.com/microsoft/onnxruntime-inference-examples ? Every file in this repo is subject to strict compliance rules, but that one doesn't because we don't ship any package from that example repo.

Sounds good to me.

**Comment by tianleiwu** [comment]
This script is quite helpful for accuracy testing in developing. Move to models/bert directory.

**Review by zhangyaobit — APPROVED** [review]
